processor/src/main/kotlin/br/com/zup/beagle/processor/utils/WidgetFileBuilder.kt

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.processor.utils
+
+import br.com.zup.beagle.annotation.ImplicitContext
+import br.com.zup.beagle.widget.context.Context
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/1767037c5777b4b44c2b78a548a8a8a8514b5d6b/processor/src/main/kotlin/br/com/zup/beagle/processor/utils/WidgetFileBuilder.kt#L20"></span>The import 'br.com.zup.beagle.widget.context.Context' is unused.</td>
    </tr>
  </tbody>
</table>